src/Umbraco.Community.FrontDoorCache/MediaSavedNotificationHandler.cs

```diff
@@ -122,13 +122,19 @@ private FrontDoorPurgeContent GetContentPathsToPurge(List<IPublishedContent> pub
                     {
                         continue;
                     }
-                    var url = media.MediaUrl(_publishedUrlProvider, mediaCulture.Key, UrlMode.Relative);
+                    var url = media.MediaUrl(_publishedUrlProvider, CultureOrNull(mediaCulture), UrlMode.Absolute);
```

Comment:
Do we want an absolute URL here? I think I remember there being config where the to clear can be set, and the list that is subsequently created is named `contentPaths`.